fract4dgui/utils.py

```diff
@@ -21,26 +21,10 @@
 break_new_things = False
```

Comment:
I think `threads_enabled`, and use in gtkfractal.py, could go then.

`break_new_things` doesn't appear anywhere else either.